api/models.py

```diff
@@ -148,15 +157,14 @@ class Meta:
         ordering = ('-created_at',)
 
     def __str__(self):
-        return "User {} interested in category {}" .format(self.user.username,
-                                                       self.follower_category.name)
+        return "User {} interested in category {}" .format(self.follower.username, self.follower_category.name)
 
 
 class Attend(BaseInfo):
```

Comment:
I think we can find a better name for this class.

Reply:
Fine by me if you have an idea of a better name. However, I think that can be done later especially on a task that is more specific and related to event attendance.